cmd/main.go

```diff
@@ -406,6 +406,7 @@ func main() {
 	go func() {
 		pullSecretUpdatesChannel := make(chan event.GenericEvent)
 
+		//TODO: remove ImageReconciler and ClusterReconciler
```

Comment:
Is there a Jira task that we could link to? So that it can be prioritized.

Reply:
I'll open one.
But for now, you can track it with https://github.com/openshift/assisted-service/pull/1112, which will get marked only after the installEnv controller PR (it is currently rebased on top of this PR)